src/bootstrap/src/core/sanity.rs

```diff
@@ -34,7 +34,6 @@ pub struct Finder {
 // Targets can be removed from this list once they are present in the stage0 compiler (usually by updating the beta compiler of the bootstrap).
 const STAGE0_MISSING_TARGETS: &[&str] = &[
     // just a dummy comment so the list doesn't get onelined
-    "wasm32-wali-linux-musl",
```

Comment:
cc @jieyouxu is this the expected thing for me to do? running `x test tests/ui` was complaining about this being present

Reply:
That seems right yes